### PR TITLE
Implemented a simple springboot database application

### PR DESCRIPTION
**Pull request by SeifMostafaa**
simple springboot application using mysql with simple crud operations

### CODE REVIEW

**Review by adriannecula — CHANGES_REQUESTED** [review]
implement review fidings

**Review by SeifMostafaa — COMMENTED** [review]
updated .gitignore file to exclude all unwanted files

**Review by adriannecula — APPROVED** [review]

**Review by gunjanmimo — APPROVED** [review]
LGTM

File: helloworld/com/example/demo/Notabarista.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "notabarista")
+@Data
+public class Notabarista implements Serializable {
```
Comment (alexandru-damian) [inline helloworld/com/example/demo/Notabarista.java:14]: Why do we have two Notabarista classes?

**Review by alexandru-damian — CHANGES_REQUESTED** [review]
Binaries, shouldn't be in this repository. Are those .class files required to be added?

**Pull request closed without merging**